pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -905,6 +905,11 @@ func (p *criStatsProvider) addCadvisorContainerStats(
 	if memory != nil {
 		cs.Memory = memory
 	}
+
+	swap := cadvisorInfoToSwapStats(caPodStats)
```

Comment:
What happens if someone turns the feature gate off? I don't see any guards for tracking stats for swap?

Reply:
You're right, but I think this ship has sailed TBH. This PR just fixes a bug of the logic already presented a while ago as part of https://github.com/kubernetes/kubernetes/pull/118865. Will it make sense to add these guards now? If so, can I do that in a follow-up after the bug is fixed?

Reply:
Yea I saw the same thing.. 

We can always fix bugs in beta.

Reply:
@haircommander WDYT?

I'm fine with a followup since most of the stats is not guarded by a feature gate for swap.

Reply:
+1 on follow up!